src/utils/shared.test.ts

```diff
@@ -1,34 +1,41 @@
 import { getChainId, WrappedOrSignedOpenAttestationDocument } from "./shared";
-import ebl from "../test/fixture/ebl.json";
-import invoice from "../test/fixture/v3/invoice-ropsten.json";
-import v2DID from "../test/fixture/sample-dns-did-verified.json";
-import v3DID from "../test/fixture/v3/dns-did-signed.json";
+import invoiceV2 from "../test/fixture/goerli/v2/invoice.json";
+import invoiceV3 from "../test/fixture/goerli/v3/invoice.json";
+import v2DID from "../test/fixture/did/dns-did-verified.json";
+import v3DID from "../test/fixture/did/dns-did-signed.json";
 
 describe("getChainId", () => {
   it("should return the correct chainId for v2 document", () => {
     const document = {
-      ...ebl,
-      data: { ...ebl.data, network: { chain: "ETH", chainId: "3" } },
+      ...invoiceV2,
+      data: { ...invoiceV2.data, network: { chain: "ETH", chainId: "3" } },
     } as unknown as WrappedOrSignedOpenAttestationDocument;
     expect(getChainId(document)).toStrictEqual(3);
```

Comment:
i think can change chainId to 5 or something else. cos i think near future we should be removing ropsten testnet.

Reply:
ok updated